release/release_workflow.wdl

```diff
@@ -0,0 +1,280 @@
+task do_release {
+   # Repo to release
+   String repo
+   # Version number that should be released
+   String releaseV
+   # Next version number after this released is done (in preparation for next release)
+   String nextReleaseV
+   # Command that will update the appropriate file for the current release
+   String updateVersionForCurrentRelaseCommand
+   # Command that will update the appropriate file for the next release
+   String updateVersionForNextRelaseCommand
+   # A command to optionnally create a fat jar
+   String? assembleJarCommand
+   
+   # Commands that will update previously released/pubslished dependencies in this repo
+   Array[String] dependencyCommands = []
+   
+   # When true, nothing will be pushed to github, allows for some level of testing
+   Boolean dryRun = false
+   
+   # Can be swapped out to try this on a fork
+   String organization = "broadinstitute"
+    
+   command {
+     set -e
+     set -x 
+     
+     # Clone repo and checkout develop
+     git clone https://github.com/${organization}/${repo}.git
+     cd ${repo}
+     git checkout develop
+     git pull --rebase
+
+     # Extract current version  
+     currentV=$(grep "git.baseVersion " build.sbt | cut -d "=" -f 2 | tr -d \"\,[:space:])
+
+     echo "Updating ${repo} to ${releaseV}"
+     ${updateVersionForCurrentRelaseCommand}
+     
+     echo "Update dependencies"
+     ${sep='\n' dependencyCommands}
+     
+     # Make sure it compiles
+     sbt update
+     sbt compile
+     
+     git add .
+     # If there is nothing to commit, git commit will return 1 which will fail the script.
+     # This ensures we only commit if build.sbt was effectively updated
+     git diff-index --quiet HEAD || git commit -m "Update ${repo} version from $currentV to ${releaseV}"
+       
+     # wdl4s needs a scala docs update
+     if [ ${repo} == "wdl4s" ]; then
+       sbt 'set scalacOptions in (Compile, doc) := List("-skip-packages", "better")' doc
+       git checkout gh-pages
+       mv target/scala-2.11/api ${releaseV}
+       git add ${releaseV}
+       git commit -m "API Docs"
+       git push ${true="--dry-run" false="" dryRun} origin gh-pages
+       git checkout develop
+       sed -i '' s/$currentV/${releaseV}/g README.md
```

Comment:
This is a touch dangerous, in case we have things like "this only works in cromwell v24". 

In fact, if `currentV` is a simple number, any instance of (e.g.) '24' in the entire README will be FTFY'd...